packages/core/src/model.ts

```diff
@@ -454,6 +454,16 @@ export class ChartModel {
 		return this.colorScale;
 	}
 
+	getColorForGroup(group: any, key?: any, data?: any) {
+		const options = this.getOptions()
+		const shouldUseFillColorForGroup = options.getFillColor && !options.getStrokeColor
+		if (shouldUseFillColorForGroup) {
+			return this.getFillColor(group, key, data)
+		} else {
+			return this.getStrokeColor(group, key, data)
```

Comment:
`getFillColor` is used in places where we'd like to determine the `fill` value of an element, it'd be misleading if the `fill` value is set by a stroke color value that's being provided. We probably don't need this line.

![image](https://user-images.githubusercontent.com/14989804/95225964-66896280-07ca-11eb-957b-906d669bf8ab.png)

Reply:
Makes sense, I realized this after opening the PR that in the cases where it would need to use the Stroke Color and the chart options had a fillColor defined it wouldn't really work, any ideas on how to overcome this?